main.py

```diff
@@ -0,0 +1,124 @@
+import sys
+class CategoryManager:
+
+    def __init__(self):
+        self.file_name = "CategoriesList.txt"
+
+    def check_file(self):
+         #check if file exists
+        try:
+            text = open(self.file_name, "r+")
+            print("Opened correctly.")
+            r_categories = text.read()
+            categories = r_categories.split( "\n" )
+            text.close()
+            return categories
+        except (IOError):
+            print("File not found.")
+
+    def categories(self):
+        pass
+
+    def add_category(self):
+        while True:
+            category_name = raw_input( "Enter a new category" )
+            if category_name in self.check_file():
+                print("Category already exist")
+                break
```

Comment:
Are you sure, that if category exists you want to break the loop, otherwise continue?

Reply:
No, I am not sure :D